src/modules/display_data.js

```diff
@@ -0,0 +1,15 @@
+export const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/SiR96bAia8ODy3OSMi9e/scores';
+
+export const scoreUl = document.querySelector('.scores');
+
+export default async function displayData() {
```

Comment:
Nice job your application is working as expected 👍. Let's convert this function into an `arrow function` you can refer to [here](https://www.digitalocean.com/community/tutorials/js-async-functions#async-arrow-function) for how to do so. Don't forget to export it as `const` and import it as `import { displayData, url, scoreUl } inside `index.js` file.